GiniVision/Classes/Core/DocumentPickerCoordinator.swift

```diff
@@ -67,8 +98,31 @@ internal final class FilePickerManager: NSObject {
         if !device.isIpad {
             setStatusBarStyle(to: .default)
         }
-
-        from.present(documentPicker, animated: true, completion: nil)
+        
+        viewController.present(documentPicker, animated: true, completion: nil)
+    }
+    
+    func showErrorDialog(for error: Error, from viewController: UIViewController) {
```

Comment:
You really need a new class/component that creates these error dialogs. Almost the same code is spread all over the code.

Reply:
I already extracted the logic of creating an alertviewcontroller to another class (`errorDialog` method), so know I only need to specify what is the message and the actions.